user/id.go

```diff
@@ -82,6 +82,11 @@ func (x ID) WriteToV2(m *refs.OwnerID) {
 // Deprecated: use [NewFromScriptHash] instead.
 func (x *ID) SetScriptHash(scriptHash util.Uint160) { *x = NewFromScriptHash(scriptHash) }
 
+// GetScriptHash gets scripthash from user ID.
+func (x ID) GetScriptHash() util.Uint160 {
```

Comment:
is used only in tests in this PR, also an additional commit?

Reply:
Moved to additional commit